build-tools/packages/build-cli/src/commands/check/policy.ts

```diff
@@ -0,0 +1,248 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import * as fs from "fs";
+import { EOL as newline } from "os";
+// eslint-disable-next-line camelcase
+import * as child_process from "child_process";
+// eslint-disable-next-line unicorn/import-style
+import * as path from "path";
+import { Flags } from "@oclif/core";
+import {
+    copyrightFileHeaderHandlers,
+    npmPackageContentsHandlers,
+    dockerfilePackageHandler,
+    fluidCaseHandler,
+    lockfilesHandlers,
+    assertShortCodeHandler,
+    Handler,
+    exclusions,
+} from "@fluidframework/build-tools";
+import { BaseCommand } from "../../base";
+
+const readPipe: () => Promise<string | undefined> = async () => {
+    return new Promise((resolve) => {
+        const stdin = process.openStdin();
+        stdin.setEncoding("utf-8");
+
+        let data = "";
+        stdin.on("data", (chunk) => {
+            data += chunk;
+        });
+
+        stdin.on("end", () => {
+            resolve(data);
+        });
+
+        if (stdin.isTTY) {
+            resolve("");
+        }
+    });
+};
+
+export class CheckPolicy extends BaseCommand<typeof CheckPolicy.flags> {
+    static description =
+        "Checks that the dependencies between Fluid Framework packages are properly layered.";
+
+    static flags = {
+        resolve: Flags.string({
+            description: `Resolve errors if possible`,
```

Comment:
double check the type for this flag. I think this should be boolean instead of string